internal/mountfuse/layout.go

```
@@ -159,21 +174,40 @@ func providerLayoutRemotePath(remoteRoot, provider string) string {
 	return joinRemotePath(joinRemotePath(remoteRoot, provider), providerLayoutFilename)
 }
 
+func legacyProviderLayoutRemotePath(remoteRoot, provider string) string {
```

golangci-lint (2.12.2)

[error] func legacyProviderLayoutRemotePath is unused

(unused)